signer/core/abihelper_test.go

```diff
@@ -45,6 +45,10 @@ func verify(t *testing.T, jsondata, calldata string, exp []interface{}) {
 
 	data, err := method.Inputs.UnpackValues(argdata)
 
+	if err != nil {
+		t.Fatal(err)
+	}
+
```

Comment:
Nitpick, please delete the two empty lines before and after the new code block :P